src/components/insights/InsightsKeyHandler.ts

```diff
@@ -31,60 +23,30 @@ type AutoDescriptionsProps = {
  * @export
  * @param {string} type
```

Comment:
This is a nitpick, but we do not need jsdoc typing when we use typescript, seems like overkill to me.